Sources/PackageRegistry/SignatureValidation.swift

```diff
@@ -203,28 +238,7 @@ struct SignatureValidation {
                 case .certificateInvalid(let reason):
                     completion(.failure(RegistryError.invalidSigningCertificate(reason: reason)))
                 case .certificateNotTrusted:
```

Comment:
Need this change: https://github.com/apple/swift-package-manager/pull/6237/files#diff-ac97ac612803fe11304731a730d285e4f7fd1cc5a2b139d3e91a50650833d58fR208

Reply:
@yim-lee sorry I merged before notifying this, hope does not cause too much trouble